_attributes/common-attributes.adoc

```diff
@@ -60,6 +60,7 @@ endif::[]
 :mtc-full: Migration Toolkit for Containers
 :mtc-version: 1.8
 :mtc-version-z: 1.8.4
+:mtc-legacy-image: 1.7
```

Comment:
it should be v1.7

Reply:
$ podman pull registry.redhat.io/rhmtc/openshift-migration-legacy-rhel8-operator:1.7
Trying to pull registry.redhat.io/rhmtc/openshift-migration-legacy-rhel8-operator:1.7...
Error: initializing source docker://registry.redhat.io/rhmtc/openshift-migration-legacy-rhel8-operator:1.7: reading manifest 1.7 in registry.redhat.io/rhmtc/openshift-migration-legacy-rhel8-operator: manifest unknown

Reply:
$ podman pull registry.redhat.io/rhmtc/openshift-migration-legacy-rhel8-operator:v1.7
Trying to pull registry.redhat.io/rhmtc/openshift-migration-legacy-rhel8-operator:v1.7...
Getting image source signatures
Checking if image destination supports signatures
Copying blob 97da74cc6d8f done   | 
Copying blob a904d427d387 done   | 
Copying blob 872602c382da done   | 
Copying blob fdaf21948019 done   | 
Copying blob d8190195889e done   | 
Copying blob a41c226c3b58 done   | 
Copying config 36bdb2f742 done   | 
Writing manifest to image destination
Storing signatures
36bdb2f74202afa5c35253df24211a1c4f9d15c1665cf34b7376ea1c652fdf76